packages/cli/src/wrappers/middleware.ts

```diff
@@ -0,0 +1,72 @@
+import { GithubClient } from '@bundler/github';
```

Comment:
shouldnt it be lower in the importing order?

Reply:
done.